desktop/nwg-displays/nwg-displays.SlackBuild

```diff
@@ -83,10 +83,8 @@ find -L . \
   -o -perm 440 -o -perm 400 \) -exec chmod 644 {} \;
 
 python3 setup.py install --optimize=1 --root=$PKG
-mkdir -p $PKG/usr/share/pixmaps $PKG/usr/share/applications
```

Comment:
you need to undelete this line as it creates the directory before you can copy something into it

Reply:
Fixed.